src/server.h

```diff
@@ -2595,10 +2594,20 @@ int serverCommunicateSystemd(const char *sd_notify_msg);
 void serverSetCpuAffinity(const char *cpulist);
 void dictVanillaFree(dict *d, void *val);
 
+/* ERROR STATS constants */
+#define ERROR_STATS_LUA_LIMIT                                                                                          \
+    128 /* After the errors RAX reaches this limit, instead of tracking                                                \
+          custom LUA errors, we track the error under the symbol below. */
+#define LUA_ERRORSTATS_OVERFLOW_ERR "LUA_ERRORSTATS_OVERFLOW"
+
 /* afterErrorReply flags */
```

Comment:
If we ever add more languages, do we want this to be only for Lua? Maybe change it to SCRIP_ERRORSTATS_OVERFLOW or just skip Lua, ERRORSTATS_OVERFLOW?

Reply:
`ERRORSTATS_OVERFLOW` sounds good to me. Will change the error message to this if others have no concerns

Reply:
That makes sense, since it does also technically apply to RM_Call.

Reply:
I updated this here: https://github.com/valkey-io/valkey/pull/500/commits/fe6e57423dcfdee5853dfc785d25191f485e484a